src/include/duckdb/common/types/value.hpp

```diff
@@ -196,6 +197,10 @@ class Value {
 private:
 	SQLType sql_type = SQLType(SQLTypeId::INVALID);
 
+	//! The hexadecimal value of the object used only for BLOB values
+	//! It's need for heap allocation
+	string hexa_str_value;
```

Comment:
Please remove this variable, I don't think it is necessary. We need to be careful when adding new variables to the Value class (and Vector/DataChunk classes).